era5_land_forcing_data2021/README.md

```diff
@@ -0,0 +1,2 @@
+# Clima Land forcing data  - ERA5 reanalysis data for 2021
+more details to follow.
```

Comment:
Can you also add a license/requested acknowledgments (if available)?

Reply:
I will double check what I have with @braghiere